validator/schema/releases_schema.py

```diff
@@ -0,0 +1,106 @@
+from validator import support
+from schema import Schema, Optional, And, Or, Regex, SchemaError
+from validator.schema.image_schema import IMAGE_CONTENT_SCHEMA
+from validator.schema.rpm_schema import RPM_CONTENT_SCHEMA
+
+GIT_SSH_URL_REGEX = r'((git@[\w\.]+))([\w\.@\:/\-~]+)(\.git)(/)?'
+
+ASSEMBLY_DEPENDENCIES = {
+    'rpms': [
+        And({
+            Optional(Regex(r'el\d+')): str,  # Each RHEL version can have its own
+            'non_gc_tag': str,  # Artist should provide tag they know will prevent this NVR from garbage collection.
+        })
+    ]
+}
+
+
+def releases_schema(file):
+    return Schema({
+        'releases': {
+            Optional(Or('stream', 'test', Regex(r'^art\d+$'))): {
+                'assembly': {
+                    Optional('basis'): {
+                        'brew_event': int
```

Comment:
In current Doozer implementation, the member of `basis` is either `brew_event` or another `assembly`

Reply:
Good catch. Added.